doc/src/records/report.md

```diff
@@ -196,6 +196,10 @@ Specifies meter(s) to be reported, for *rpType*=MTR.
   ----------- ----------------------------- ------------- --------------------------- -----------------
               name of a *METER*, ALL, SUM                 Required for *rpType*=MTR   constant
 
+**rpAH**
```

Comment:
Did you see this included already in the conditional statement below?

Reply:
It is if-out should I remove the if statement and replace it, or should I keep it how it is right now?

Reply:
Similarly here. What is the status on this thread?